src/index.html

```diff
@@ -5,8 +5,402 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="style-header">
+      <a href="#Home.html">
+        <img 
+          class="logo" 
+          src="images/logo.png"
+          alt="MoyoLogo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item active-item">
+            <a class="nav__link nav__link--is-active" href="#apple">Apple</a> 
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a> 
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              class="nav__link" 
+              href="#Laptops & Computers" 
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="card-container">
+        <div class="card card-container__item" data-qa="card">
+          <img 
+            class="card__image image" 
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock" 
+          >
+          
+          <h3 class="card__device-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+    
+          <p class="card__device-code">Product code: 195434</p>
+    
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+            </div>
+    
+            <p class="card__text">Reviews: 5</p>
+          </div>
+    
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+    
+            <p class="card__price-price_2199">$2,199</p>
+          </div>
+    
+          <a 
+            class="card__link" 
+            href="/" 
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card card-container__item" data-qa="card">
```

Comment:
separate blocks with empty space
```suggestion
        </div>

        <div class="card card-container__item" data-qa="card">
```